src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewController.swift

```diff
@@ -262,10 +262,6 @@ extension DynamicTableViewController {
 			if !isLast { cell.addSeparator(.inBetween) }
 		}
 
-		if let cellBackgroundColor = cellBackgroundColor {
-			cell.backgroundColor = cellBackgroundColor
-		}
-
```

Comment:
if cellBackground gets no longer set over here - do we still need the property on top
```
	@IBInspectable var cellBackgroundColor: UIColor? = .clear
```
There are at least 6 classes using that and now it won't get set to the view? I Thinkl all of them use .clear color, so this might be a good moment to remove the property, use .clear as a default color and remove setting the property inside the classes as well?

Reply:
I agree, will do that